app/src/main/java/io/intrepid/skeleton/base/BaseView.java

```diff
@@ -0,0 +1,4 @@
+package io.intrepid.skeleton.base;
+
+public interface BaseView {
```

Comment:
You can have BaseActivity and BaseFragment implement this interface, add fun methods like ` BaseActivity getBaseActivity();` `Bundle getArguments();` so that presenters can get casted "View" context and arguments. I assume they were not added because this might not be how we want to implement MVP